app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/models/Order.java

```diff
@@ -1,16 +1,15 @@
 package com.epfl.neighborfood.neighborfoodandroid.models;
 
-import java.io.Serializable;
 import java.util.Date;
 
-public class Order implements Serializable {
+public class Order extends Model {

```

Comment:
good thing that all the models now inherit the abstract class model like we discussed